terraform/modules/dbx_tables_access/dbx_grant_table_access/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "ad_group_names" {
+  description = "Set of AD group names to grant access to the catalog"
+  type        = set(string)
+}
+
+variable "full_table_name" {
+  description = "Name of the table to grant access to"
+  type        = string
+}
+
+variable "table_privileges" {
+  description = "List of privileges to grant on the table. Can be APPLY_TAG, MODIFY, SELECT, ALL_PRIVILEGES, or a combination of these"
+  type        = list(string)
```

Comment:
Kanskej denne her er et godt case for å bruke terraform validation i en variabel? Eller får du kanskje feilmelding hvis du prøver å legge inn feil type her uansett?

```hcl
validation {
    condition     = contains(["a", "b", "c"], var.input_parameter)
    error_message = "Allowed values for input_parameter are \"a\", \"b\", or \"c\"."
}
```

Reply:
Tror du får feil type uansett?